spec/support/codeowners_parser.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CodeownersParser
+  def perform(team_name)
+    parsed_codeowners = File.read('.github/CODEOWNERS').split("\n").map do |line|
```

Comment:
do we need to filter out files that don't have specs? like spec files, config files, rakefiles, etc?

Reply:
That's already happening automatically, of our spec files, config files, and rake files the only one that appears in the OctoIdentity SimpleCov group is `user_credential.rake` - which is as far as I know the only one of our rake files that has a dedicated spec, `user_credential_test.rb`.

The overall SimpleCov is configured to `track_files '**/{app,lib}/**/*.rb'` so it appears to ignore anything not in those two directories unless they're specifically added like all of the modules are or somehow picked up intelligently like it did with the `user_credential.rake` file.

Reply:
cool sounds good